### DISCUSSION TITLE
Use Appropriate Solder type

### DISCUSSION THREAD

**Discussion by Landsat-Mike** [STELLA-Landsat/STELLA#32] [Notes from the Field]
I used solder that wasn't on the recommended list. It was still lead free but had more flux than the listed solder and those STELLAs over time have had some corrosion problems. We cleaned the corrosion with some alcohol and q-tips. It taught me a good lesson which is to adhere to the recommended parts list and ask the engineer (Paul) before any deviation.

**Comment by paulmirel** [comment]
Slight correction. It wasn't that the solder had more flux. It's that the flux in the off brand solder left corrosive residue that lead to copper sulfate forming on, and interfering with, the joints. Buy your solder from an electronics supplier in the US, not from a general purpose consumer products company.
